index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project list and about section</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="canvas">
```

Comment:
- [ ] Kindly make sure the container of the section is a `section` tag 👍🏽
- [ ] Please note that the main container should have the `display: grid`property.

Reply:
I understood your point, you can remove the `div` tags and start with a section tag with the class `project-list` as you already have 👍. Please don't forget make this the continuation of your `add toolbar and introduction` milestone. 💪🏽

Reply:
Already done thank you.